lib/linter.coffee

```diff
@@ -0,0 +1,147 @@
+{BufferedProcess, CompositeDisposable} = require 'atom'
+helpers = require 'atom-linter'
+fs = require 'fs'
+fse = require 'fs-extra'
+path = require 'path'
+temp = require 'temp'
+XRegExp = require('xregexp').XRegExp
```

Comment:
Looks like that was added after I wrote this (and will require a version bump in the dependencies).  I will work on refactoring this to use the helper module.